### DISCUSSION TITLE
how to preserve layout after exiting Fullscreen video mode

### DISCUSSION THREAD

**Discussion by vishal340** [i3/i3#5823] [Q&A]
let's say I have 2 browser(edge) windows in a workspace with horizontal split. I go fullscreen mode on one window. then I open a video in that window with fullscreen mode. when I exist the fullscreen mode of video, the layout of the workspace goes back to the horizontal  split mode instead of the fullscreen mode of the window I was using just before it.
how can I fix this issue? is this intended to work this way?
I just started using i3 few days ago.
thanks in advance.

**Comment by orestisfl** [comment]
> is this intended to work this way?

Yes, there is only one fullscreen window in i3, once it stops existing or if fullscreen mode is disabled, no other window is automatically switched to fullscreen mode again.
